knowledge-base/calendar-disable-weekends.md

```diff
@@ -0,0 +1,104 @@
+---
+title: Disable Weekends in Calendar
+description: How to disable Weekends in Calendar?
+type: how-to
+page_title: Disable Weekends in Calendar
+slug: calendar-kb-disable-weekends
+position: 
+tags: disable, weekends, calendar, datepicker, daterangepicker
+ticketid: 1543661
+res_type: kb
+---
+
+## Environment
+<table>
+	<tbody>
+		<tr>
+			<td>Product</td>
+			<td>Calendar for Blazor, DatePicker for Blazor, DateRangePicker for Blazor</td>
+		</tr>
+	</tbody>
+</table>
+
+
+## Description
+I want to disable the weekends in the Calendar Month view, so the user is not able to select them.
+
+
+## Solution
+
+You can create a collection of weekend days and pass it to the [Disabled Dates]({%slug components/calendar/selection%}#disabled-dates) parameter of the Calendar.
```

Comment:
Adding the "prevent" keyword, as we don't have it anywhere.

```suggestion
You can create a collection of weekend days and pass it to the [Disabled Dates]({%slug components/calendar/selection%}#disabled-dates) parameter of the Calendar. This will prevent the user from selecting them.
```